packages/web/src/components/spa/HackAUBG/JudgesSection/JudgesSection.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import axios from 'axios';
+import { useState } from 'react';
+import { useEffect } from 'react';
+import { url } from '../../../../Global';
+import { JudgesCard } from './JudgesCard';
+import './judges_section.css';
+
+export const JudgesSection = () => {
+    const [jury, setJury] = useState([{}]);
+
+    const getJury = () => {
+        axios({
+            method: 'get',
+            url: url + '/api/jury'
+        })
+            .then((res) => {
+                setJury(res.data.data.data);
+            })
+            .catch((err) => {
+                console.log(err);
+            });
+    };
+
+    const handleClick = (event) => {
+        return () => {
+            window.open(event, '_blank', 'noopener noreferrer');
+        };
+    };
+
+    const renderMap = () => {
+        if (jury) {
+            return (
+                <div className="judges-container">
+                    <h1 className="judges-header">Judges</h1>
+                    <div className="judges-picture">
+                        {jury.map((judge, index) => (
+                            <div key={index} className="judge-div">
+                                <JudgesCard
+                                    Judge={judge}
+                                    handleClick={handleClick(judge.sociallink)}
```

Comment:
In this case you're not doing anything with the event (handling the passed event), so therefore it's preferable if you use the onClick hook https://stackoverflow.com/a/60048346